unison-src/transcripts/kind-inference.md

```diff
@@ -54,6 +54,13 @@ unique type T a = T a
 unique type S = S (T Nat)
 ```
 
+Delay kind defaulting until all components are processed
+```unison:error
+unique type T a = T
+
+unique type S = S (T Optional)
+```
+
```

Comment:
Could you add a little extra explanation about what this stanza is showing?
I'm also confused because although it's marked as error, I don't see an error in the output?

Reply:
Sure, the `unison:error` was accidentally left from the preceding commit, when this was an error. I removed the `error` and added a comment.

Reply:
Got it, thanks!